block-producer/src/block_builder/prover/mod.rs

```diff
@@ -133,10 +190,30 @@ impl BlockProver {
             execute(&self.kernel, stack_inputs, host, ExecutionOptions::default())
                 .map_err(BlockProverError::ProgramExecutionFailed)?;
 
+        // TODO: Use `StackOutputs::pop_digest()` once merged
+        let (account_root_output, note_root_output) = {
```

Comment:
There are only a few nits left in https://github.com/0xPolygonMiden/miden-vm/pull/1155 - so, we could probably merge that and then update this PR.